util/feature.go

```diff
@@ -0,0 +1,79 @@
+package util
+
+// Feature custom events
+var featureCustomEvents bool
```

Comment:
Can't we have common setters and getters, we are just replicating code which is not required.